.github/CHANGELOG.md

```diff
@@ -4,9 +4,11 @@
 
 <h3>Improvements 🛠</h3>
 
-- Added comprehensive tests for Python bindings.
+- Replaced pybind11 bindings with nanobind
+  [(#1176)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1176)
   [(#1198)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1198)
-  
+  [(#1184)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1184)
```

Comment:
Could you please specify which bindings are replaced where (in terms of PRs.)?
You should probably break this down in different entries.

Reply:
How are the changes in a5e0279 ?